WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -811,20 +815,32 @@ private extension GetStartedViewController {
 
         buttonViewController.hideShadowView()
 
-        // Add a "Continue" button here as the `continueButton` at the top
-        // will not be displayed for `signInUsingSiteCredentials` screen mode.
-        //
-        buttonViewController.setupTopButton(title: ButtonConfiguration.Continue.title,
-                                            isPrimary: true,
-                                            accessibilityIdentifier: ButtonConfiguration.Continue.accessibilityIdentifier) { [weak self] in
-            self?.handleSubmitButtonTapped()
-        }
+        if configuration.enableSocialLogin {
+            configureSocialButtons()
 
-        // Setup Sign in with site credentials button
-        buttonViewController.setupBottomButton(attributedTitle: WPStyleGuide.formattedSignInWithSiteCredentialsString(),
-                                               isPrimary: false,
-                                               accessibilityIdentifier: ButtonConfiguration.SignInWithSiteCredentials.accessibilityIdentifier) { [weak self] in
-            self?.handleSiteCredentialsButtonTapped()
+            // Setup Sign in with site credentials button
+            buttonViewController.setupTertiaryButton(attributedTitle: WPStyleGuide.formattedSignInWithSiteCredentialsString(),
+                                                   isPrimary: false,
+                                                   accessibilityIdentifier: ButtonConfiguration.SignInWithSiteCredentials.accessibilityIdentifier) { [weak self] in
+                self?.handleSiteCredentialsButtonTapped()
+            }
+        } else {
+            // Add a "Continue" button here as the `continueButton` at the top will be hidden
+            //
+            if showsContinueButtonAtTheBottom {
+                buttonViewController.setupTopButton(title: ButtonConfiguration.Continue.title,
+                                                    isPrimary: true,
+                                                    accessibilityIdentifier: ButtonConfiguration.Continue.accessibilityIdentifier) { [weak self] in
+                    self?.handleSubmitButtonTapped()
+                }
+            }
+
+            // Setup Sign in with site credentials button
+            buttonViewController.setupBottomButton(attributedTitle: WPStyleGuide.formattedSignInWithSiteCredentialsString(),
+                                                   isPrimary: false,
+                                                   accessibilityIdentifier: ButtonConfiguration.SignInWithSiteCredentials.accessibilityIdentifier) { [weak self] in
+                self?.handleSiteCredentialsButtonTapped()
+            }
```

Comment:
Maybe we can simplify this for readability:

```Swift
if configuration.enableSocialLogin {
    configureSocialButtons()
} else {
    buttonViewController.setupTopButton(title: ButtonConfiguration.Continue.title, 
                                        isPrimary: true, 
                                        accessibilityIdentifier: ButtonConfiguration.Continue.accessibilityIdentifier) { [weak self] in
        self?.handleSubmitButtonTapped()
    }
}


// Setup Sign in with site credentials button
buttonViewController.setupBottomButton(attributedTitle: WPStyleGuide.formattedSignInWithSiteCredentialsString(), 
                                       isPrimary: false, 
                                       accessibilityIdentifier: ButtonConfiguration.SignInWithSiteCredentials.accessibilityIdentifier) { [weak self] in
     self?.handleSiteCredentialsButtonTapped()
}
```

Reply:
The site credentials button needs to be set in the tertiary button when social login is enabled in config. (It is not always the bottom button) So the above code won't work. 

Also, keeping the `showsContinueButtonAtTheBottom` boolean helps to maintain code readability in other places.